### PR TITLE
Read product

### PR DESCRIPTION
**Pull request by va1shn9v**
-  Modified Create Product to Match DB model
- Added additional test coverage
- Added Read API

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -55,21 +73,81 @@ def test_index(self):
         resp = self.client.get("/")
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
 
+    def test_create_pet(self):
```
Comment (tawn33y) [inline tests/test_routes.py]: **Nit:** found a few instances with "create pet" instead of "create product"

File: tests/test_routes.py [hunk 2]
```diff
@@ -55,21 +73,81 @@ def test_index(self):
...
-        self.assertIn('already exists', response.data.decode())
-
-    def test_create_product_invalid_data(self):
-        invalid_data = self.product_data.copy()
-        del invalid_data['price']
-        response = self.client.post('/products', data=json.dumps(invalid_data), content_type='application/json')
-        self.assertEqual(response.status_code, 400)
-        self.assertIn('Invalid data', response.data.decode())
\ No newline at end of file
+    # def test_create_product_success(self):
```
Comment (tawn33y) [inline tests/test_routes.py:123]: **Nit:** do you think we should delete this commented out code?

**Review by tawn33y — CHANGES_REQUESTED** [review]
Hi @va1shn9v,
Looks good! Left two comments that are non-blocking.

File: tests/test_routes.py (hunk 2 above)
Comment (va1shn9v) [reply tests/test_routes.py:123]: I think we can let it stay and use it to add additional code coverage

File: tests/test_routes.py (hunk 1 above)
Comment (va1shn9v) [reply tests/test_routes.py]: Will edit it and push

**Review by f0rgeAhead — COMMENTED** [review]
I have reviewed this.

**Pull request merged**